assets/js/base/context/cart-checkout/payment-methods/payment-method-data-context.js

```diff
@@ -124,10 +126,13 @@ export const PaymentMethodDataProvider = ( {
 		reducer,
 		DEFAULT_PAYMENT_DATA
 	);
-	const setActivePaymentMethod = ( paymentMethodSlug ) => {
-		setActive( paymentMethodSlug );
-		dispatch( statusOnly( PRISTINE ) );
-	};
+	const setActivePaymentMethod = useCallback(
+		( paymentMethodSlug ) => {
+			setActive( paymentMethodSlug );
+			dispatch( statusOnly( PRISTINE ) );
+		},
+		[ setActive, dispatch ]
+	);
```

Comment:
Memoizing things in the context is more important (especially for nested context providers like we're doing) to limit the amount of `useRef` usage which is code smell.